packages/components/src/DataList/components/DataListHeaderTile/DataListHeaderTile.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import classnames from "classnames";
+import styles from "./DataListHeaderTile.css";
+import { DataListHeader, DataListObject } from "../../DataList.types";
+import { Text } from "../../../Text";
+import { Icon } from "../../../Icon";
+import { useDataListContext } from "../../context/DataListContext";
+
+interface DataListHeaderTileProps<T extends DataListObject> {
+  headers: DataListHeader<T>;
+  headerKey: string;
+}
+
+export function DataListHeaderTile<T extends DataListObject>({
+  headers,
+  headerKey,
+}: DataListHeaderTileProps<T>) {
+  const { sorting } = useDataListContext();
+
+  const isSortable = sorting?.sortable.includes(headerKey);
+  const sortingState = sorting?.state;
+
+  function toggleSorting(sortingKey: string) {
+    if (sortingKey === sortingState?.key) {
+      sorting?.onSort({
+        direction: sortingState.direction === "asc" ? "desc" : "asc",
+        key: sortingKey,
+      });
```

Comment:
i wonder if you remove the rest and keep this, would it still work the same?

Reply:
lol it does work 🤦 thanks for catching it